lib/bootloader_setup.pm

```diff
@@ -99,48 +100,54 @@ And of course the new entries have C<ima_policy=tcb> added to kernel parameters.
 
 =cut
 sub add_custom_grub_entries {
-    my $grub_param = get_var('GRUB_PARAM');
-    return unless defined($grub_param);
+    my @grub_params = split(/;/, get_var('GRUB_PARAM', ''));
```

Comment:
```suggestion
    my @grub_params = split(/\s*;\s*/, trim(get_var('GRUB_PARAM', '')));
```
using https://mojolicious.org/perldoc/Mojo/Util#trim

Reply:
@cfconrad thanks for a review. IMHO trim is not needed (everything works the same with extra spaces), but ok, I'll add it. 

I'd like to get this + https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9217 merged, I consider poo#61203 (and poo#51743 which is also partly affected by this) as a high priority.